lisa/tools/swap.py

```diff
@@ -14,6 +14,14 @@ def command(self) -> str:
     def _check_exists(self) -> bool:
         return True
 
+    def set_up_swap_space(self, partition: str) -> None:
```

Comment:
how about `create_swap`?

Reply:
`partition_name` is clearer. But where the actual disk space is from?

Reply:
Current logic is adding data disk.
As discussed we may implement Matt and Divyansh's change later